pkg/cli/image/manifest/errors.go

```diff
@@ -67,3 +67,10 @@ func IsImageNotFound(err error) bool {
 		return false
 	}
 }
+
+func IsFilteredAllImagesFromManifestList(err error) bool {
```

Comment:
This hurts readability. We don't need to wrap just 3 lines of code into a function.

Reply:
I will update this. I made the change due to some other feedback I had received.